crt_portal/static/sass/custom/confirmation.scss

```diff
@@ -110,7 +110,7 @@ section hr {
 section.what-to-expect {
   @include u-bg($theme-color-primary-lightest);
   @include u-margin-top(5);
-  padding-bottom: 14rem;
+  padding-bottom: 4rem;
```

Comment:
assuming that this was intentional and not a typo? (just asking 'cause `4` and `14` are so close)

Reply:
Intentional! @Jacklynn wants this gap reduced.